docs/docs/developer-guides/cosmwasm-cli.md

```diff
@@ -1,35 +1,53 @@
 # Composable CosmWasm CLI
 
-Composable Cosmwasm CLI is a CLI tool to quickly get started with the XCVM ecosystem and interact with a chain that runs `pallet-cosmwasm`. In this guide, we will show you how to run the CLI on Picasso rococo.
+Composable Cosmwasm CLI is a CLI tool to quickly get started with the XCVM ecosystem and interact with a chain that runs `pallet-cosmwasm`. In this guide, we will show you how to run the CLI on a local Picasso network and Picasso Rococo. 
 
-## Installing `ccw`
+:::info 
+Picasso Rococo is a testnet (test network) for [Picasso](../parachains/picasso-parachain-overview.md). It allows developers to experiment, test runtime module deployment, and refine their applications to ensure the stability and compatibility of new features before deploying on Picasso mainnet by interacting with the [Rococo Relay Chain](https://polkadot.network/blog/rococo-revamp-becoming-a-community-parachain-testbed/).
+:::
 
-There are two methods to installing the ccw-vm:
+## Setting up the development environemnt
+
+The process of setting up a development environment for deploying CosmWasm contracts, both a local Picasso netork and on Picasso Rococo, follows the same procedure. There is a distinction in the RPC endpoint mentioned in the CLI commands to upload, instantiate and execute contracts. To interact with a local Picasso network, you will utilize `http://127.0.0.1:32200` whereas to deploy on Picasso Rococo, you will employ `wss://picasso-rococo-rpc-lb.composablenodes.tech:<insert port>`. Additionally, please note that the "-n alice" sudo key will be substituted with your seed phrase when entering the commands.
+
+Nix is a requirement to set up and start a local development environment with Composable's code. We recommend using the [Zero-to-Nix installer](https://zero-to-nix.com/start/install) to install Nix. Refer to our [docs](../nix.md) for more information.
+
+### Installing `ccw`
+
+There are two methods to installing the `ccw-vm`:
 
 1. At first, clone the [Composable moonorepo](https://github.com/ComposableFi/composable):
 
 ```
 git clone https://github.com/ComposableFi/composable
 ```
 
-Then run the following command to install the ccw binary:
+Then run the following command to install the `ccw` binary:
 
 ```
 cargo install --path ./composable/code/parachain/frame/cosmwasm/cli
 ```
 
-2. An alternative method to run the ccw-vm is by running the following command which requires Nix. Nix is required to start the development environment to deploy contracts on Picasso rococo:
+2. An alternative method to run the `ccw-vm` is by running the following command which utilizes Nix. Nix is a package manager used for Composable software and required to start the development environment to deploy contracts on Picasso Rococo:
```

Comment:
you do not need to write english. 

just 

## Install

### Cargo

``
...cargo commands


### Nix Falkes

```
nix command...
```

do not write utilises required, be short.

do not explain what nix or cargo here.